### PR TITLE
[Doctest] Add configuration_realm.py

### PR DESCRIPTION
**Pull request by ak04p**
Add configuration_realm.py to utils/documentation_tests.txt for doctest.
Based on issue #19487 

@ydshieh could you please check it?
Thank you

### CODE REVIEW

**Comment by ak04p** [comment]
Thank you for the feedback, I'll correct it.

File: src/transformers/models/realm/configuration_realm.py [hunk 1]
```diff
@@ -116,16 +116,17 @@ class RealmConfig(PretrainedConfig):
...
     >>> # Initializing a REALM realm-cc-news-pretrained-* style configuration
     >>> configuration = RealmConfig()
 
-    >>> # Initializing a model from the google/realm-cc-news-pretrained-embedder style configuration
+    >>> # Initializing a model (with random weights) from the google/realm-cc-news-pretrained-embedder style configuration
     >>> model = RealmEmbedder(configuration)
 
     >>> # Accessing the model configuration
     >>> configuration = model.config
+
```
Comment (ydshieh) [inline src/transformers/models/realm/configuration_realm.py:129]: no need this empty line

File: utils/documentation_tests.txt [hunk 2]
```diff
@@ -139,3 +139,4 @@ src/transformers/models/yolos/configuration_yolos.py
 src/transformers/models/yolos/modeling_yolos.py
 src/transformers/models/x_clip/modeling_x_clip.py
 src/transformers/models/yoso/configuration_yoso.py
+src/transformers/models/realm/configuration_realm.py
```
Comment (ydshieh) [inline utils/documentation_tests.txt:142]: This should be in an ordered place 🙏

**Review by ydshieh — APPROVED** [review]
Thank you @ak04p .

The original empty line at the end of `utils/documentation_tests.txt` was removed, but I added it back.

**Review by ydshieh — APPROVED** [review]

**Pull request merged**